contracts/renderers/CrowdfundNFTRenderer.sol

```diff
@@ -338,7 +338,7 @@ contract CrowdfundNFTRenderer is RendererBase {
     function getCrowdfundType() private view returns (CrowdfundType) {
         // If this function does not exist, then it is an NFT crowdfund.
         (bool success, bytes memory res) = address(this).staticcall(
-            abi.encodeWithSignature("exchangeRateBps()")
+            abi.encodeWithSignature("exchangeRate()")
```

Comment:
backwards compatibility issue

Reply:
I'm going to merge #352 which solves this issue. This doesn't need to be here since `ReraiseETHCrowdfund` isn't a consideration anymore.

Reply:
Isn't it for an ETH crowdfund vs NFT crowdfund which isn't related to `ReraiseETHCrowdfund`?

Reply:
They're related if you read the contact. There is no concept of a crowdfund NFT for ETH Parties without the `ReraiseETHCrowdfund`.